src/main/react/components/dialog/DialogFooter.js

```diff
@@ -1,17 +1,35 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { Button, DialogActions } from "@material-ui/core";
+import { Box, Button, DialogActions } from "@material-ui/core";
+import CircularProgress from "@material-ui/core/CircularProgress";
 
 export function DialogFooter({
   formId,
   onClose,
   onSubmit,
   onDelete,
+  onExport,
   disableDelete = false,
   disableEdit = false,
+  processingExport = false,
 }) {
   return (
     <DialogActions>
+      {onExport && (
+        <Button disabled={processingExport} onClick={onExport} tabIndex={1}>
```

Comment:
Why does this have a tabindex? Besides: you rarely (close to never) want to use a tabindex greater than 0. This is because you are manually overriding the tab index and this leads to unexpected situations when tabbing.